tests/test_sequence_tagging.py

```diff
@@ -0,0 +1,57 @@
+from tempfile import TemporaryDirectory
+from typing import Dict, List, Any, Tuple
+
+import datasets
+import flair
+import numpy as np
+import pytest
+from flair.data import Corpus
+
+from embeddings.data.hugging_face_data_loader import HuggingFaceDataLoader
+from embeddings.data.hugging_face_dataset import HuggingFaceDataset
+from embeddings.embedding.flair_embedding import FlairTransformerWordEmbedding
+from embeddings.evaluator.sequence_tagging_evaluator import SequenceTaggingEvaluator
+from embeddings.model.flair_model import FlairModel
+from embeddings.pipeline.standard_pipeline import StandardPipeline
+from embeddings.task.flair_task.sequence_tagging import SequenceTagging
+from embeddings.transformation.flair_transformation.column_corpus_transformation import (
+    ColumnCorpusTransformation,
+)
+from embeddings.transformation.flair_transformation.downsample_corpus_transformation import (
+    DownsampleFlairCorpusTransformation,
+)
+
+
+@pytest.fixture  # type: ignore
+def sequence_tagging_pipeline(
+    result_path: "TemporaryDirectory[str]",
+) -> Tuple[
+    StandardPipeline[str, datasets.DatasetDict, Corpus, Dict[str, np.ndarray], Dict[str, Any]],
+    "TemporaryDirectory[str]",
+]:
+    dataset = HuggingFaceDataset("clarin-pl/nkjp-pos")
+    data_loader = HuggingFaceDataLoader()
+    transformation = ColumnCorpusTransformation("tokens", "pos_tags").then(
+        DownsampleFlairCorpusTransformation(percentage=0.001)
+    )
+    embedding = FlairTransformerWordEmbedding("allegro/herbert-base-cased")
+    task = SequenceTagging(result_path.name, hidden_size=256, task_train_kwargs={"max_epochs": 1})
+    model = FlairModel(embedding, task)
+    evaluator = SequenceTaggingEvaluator()
+
+    pipeline = StandardPipeline(dataset, data_loader, transformation, model, evaluator)
+    return pipeline, result_path
+
+
+def test_sequence_tagging_pipeline(
+    sequence_tagging_pipeline: Tuple[
+        StandardPipeline[str, datasets.DatasetDict, Corpus, Dict[str, np.ndarray], Dict[str, Any]],
+        "TemporaryDirectory[str]",
+    ],
+) -> None:
+    flair.set_seed(441)
+    pipeline, path = sequence_tagging_pipeline
+    result = pipeline.run()
+    path.cleanup()
+
+    np.testing.assert_almost_equal(result["seqeval"]["overall_f1"], 0.023668639053254437)
```

Comment:
i would change that the f1 is not smaller then, so we will be able to increase the f1 but not decrease, we could log warning when it is not equal so we will be able to increase the expected value

Reply:
up or down it is still a change in result, so maybe it is better to know that something changed in the pipeline and fail not only for the worse result? wdyt?

Reply:
I think that the score should remain constant. The main goal of these tests are to check whether we are able to run whole sequence tagging pipeline. 

This test is optimized to speedup run time. Therefore there are some limitations:
-  We use only 1% of dataset
-  Model is trained only on one epoch

Random states are also set. So I don't see use cases whether we improve that score.

Reply:
:+1: